Tests/Functional/Backend/Grid/ContainerGridColumnItemTest.php

```diff
@@ -36,7 +36,10 @@ public function getNewContentUrlContainsUidOfLiveWorkspaceAsContainerParent(): v
         $pageLayoutContext->expects(self::any())->method('getPageId')->willReturn(3);
         $gridColumn = $this->getMockBuilder(GridColumn::class)->disableOriginalConstructor()
             ->getMock();
-        $containerGridColumnItem = $this->getAccessibleMock(ContainerGridColumnItem::class, ['foo'], [], '', false);
+        $containerGridColumnItem = $this->getMockBuilder($this->buildAccessibleProxy(ContainerGridColumnItem::class))
+            ->disableOriginalConstructor()
+            ->addMethods(['foo'])
```

Comment:
node addMethods() has been deprecated with phpunit v10.1, we'll start removing it from core codebase soon. just use setMethods([]) to say "don't mock-away anything".

Reply:
i will do a new PR